src/components/Accordion/AccordionItem.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { AccordionItem as CarbonAccordionItem } from 'carbon-components-react';
+
+// same default used in carbon. adding to avoid prop type validation issues.
+const defaultRenderExpando = props => <button type="button" {...props} />;
+
+const AccordionItem = ({ children, ...props }) => {
+  const [openState, setOpenState] = useState(false);
```

Comment:
will this get updated if the incomming open prop changes?  I can't really tell from the code, but I don't really see anything that would cause it to update... and should the default value be props.open instead of false?

Reply:
yes. good catch